shopinvader_search_engine/views/shopinvader_variant_view.xml

```diff
@@ -6,12 +6,10 @@
     <field name="inherit_id" ref="shopinvader.shopinvader_variant_tree_view" />
     <field name="arch" type="xml">
         <field name="lang_id" position="after">
-            <field name="index_id" />
-        </field>
-        <field name="name" position="after">
-            <field name="sync_state" />
-            <field name="date_modified" />
-            <field name="date_syncronized" />
+            <field name="index_id" optional="show" />
```

Comment:
if you hide the index, you must display the lang. Bear in mind that the project you are working on uses only one lang. I can show you one w/ 3 langs :wink:

Reply:
same as https://github.com/shopinvader/odoo-shopinvader/pull/1060#discussion_r683491827

Reply:
still, I don't get why this change is required. What are you trying to solve?

Reply:
Oh it's not _required_, It's an UX improvement :sweat_smile:. I take this is why `optional="show"` was created.

See the core's product tree for example:
https://github.com/odoo/odoo/blob/c6467305c88fb3ba3c832a4cca525220f977ba53/addons/product/views/product_views.xml#L301-L313 Pretty much all fields in there are either `optional="show"` or `optional="hide"`.

Since `optional="show" doesn't do much besides giving the user the ability to hide the column, as a rule, I tend to use this as much as possible.

TBH I don't understand _why not_, but I reverted so we can move forward